release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -650,6 +650,18 @@ If you disable the `ImageRegistry` capability or if you disable the integrated {
 
 For more information, see xref:../nodes/pods/nodes-pods-secrets.adoc#auto-generated-sa-token-secrets_nodes-pods-secrets[Automatically generated secrets].
 
+[discrete]
+[id="ocp-4-15-ovnic-default-range"]
+==== Open Virtual Network Infrastructure Controller default range
+
+Previously, the IP range `168.254.0.0/16` was the default IP used for the transit switch subnet in the Open Virtual Network Infrastructure Controller. With this update, the `100.88.0.0/16` IP range is now the default IP used. Customers should not use this IP range in their infrastructure network. (link:https://issues.redhat.com/browse/OCPBUGS-20178[*OCPBUGS-20178*])	
```

Comment:
I would go a bit more verbose, as "the IP" is not technically correct. Also active voice is preferable.

Moreover, the IBM SG says about "should": Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

So I rewrite without "should" as best I could guess - the final decision is the author's of course